scripts/install-deps.sh

```diff
@@ -561,6 +571,7 @@ main()
             ;;
     esac
 
+    fetch_and_unpack_boxed
```

Comment:
this needs to be also done for the PowerShell install-deps.ps1 script, similar to

https://github.com/contour-terminal/contour/blob/master/scripts/install-deps.ps1#L35-L38

that should at least fix the windows CI build error.